english/hulk/smash.md

```diff
@@ -3,5 +3,5 @@ As Hulk, you easily break through the floor and fall, only to find that it's tur
 You flail your arms wildly, whacking the turtles left and right, screaming "Hulk Smash!" with every broken shell. 
 
 You can't get them all though, and a particularly large ones knocks your head and you pass out.
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.